pattern-1/conf/mysql/dbscripts/is_identity.sql

```diff
@@ -22,6 +22,34 @@
 -- IDENTITY TABLES
 -- ################################
 
+USE WSO2_IDENTITY_DB;
```

Comment:
This should come under license. Looks like duplicated license header and statement is duplicated

Reply:
Fixed.